libdnf/repo/temp_files_memory.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright Contributors to the libdnf project.
```

Comment:
What about to use `dnf5 project`

Reply:
This was already discussed in some other PR, these headers are same across the codebase and should be replaced in a batch. Referenced in a separate issue: https://github.com/rpm-software-management/dnf5/issues/410